src/components/_forms/MigrationForm.tsx

```
@@ -0,0 +1,410 @@
+      // Get minimum mint amount
+      const minimumMint = await boringVaultLens?.read.previewDeposit([
+        cellarConfig.baseAsset.address,
+        amountOfBaseAsset,
+        alphaStEth.config.cellar.address,
+        alphaStEth.config.accountant?.address,
+      ])
+
+      // Execute migration
+      const migrationHash = await writeContractAsync({
+        address: cellarSigner?.address as `0x${string}`,
+        abi: cellarSigner?.abi!,
```

ESLint

[error] Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.

(@typescript-eslint/no-non-null-asserted-optional-chain)